README.md

```diff
@@ -1,2 +1,137 @@
-# School_Library_Manager
-This is a School library management system
+# ENUMERABLE
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [ENUMERABLE] <a name="about-project"></a>
+
+**[ENUMERABLE]** The Enumerable mixin provides collection classes with several traversal and searching methods, and with the ability to sort. The class must provide a method each, which yields successive members of the collection.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">RUBY</a></li>
+  </ul>
+
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Test the same funcionality as methods in Enumerable]**
+
+### Live Demo <a name="live-demo"><a/>
+
+[Live Demo Link]()
+
+### Video <a name="video"><a/>
+
+[Video Link]()
```

Comment:
Great job @Mukumbuta 🤝

- [ ] Kindly remove the live demo and video links from your readme file since this project is not deployed and there is no video presentation when the video link is clicked.